webpack/assets/javascripts/react_app/components/UserJwtToken/components/UserJwtTokenForm.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { Grid } from 'patternfly-react';
+import PropTypes from 'prop-types';
+
+import FormField from '../../common/forms/FormField';
+import Select from '../../common/forms/Select';
+import { translate as __ } from '../../../common/I18n';
+
+const UserJwtTokenForm = ({
+  expValue,
+  setExpValue,
+  expUnit,
+  setExpUnit,
+  handleSubmit,
+}) => {
+  const units = [
+    { label: __('Never'), value: 'never' },
+    { label: __('Hour'), value: 'hours' },
+    { label: __('Day'), value: 'days' },
+    { label: __('Week'), value: 'weeks' },
+    { label: __('Month'), value: 'months' },
+  ];
+
+  return (
+    <div>
+      <Grid.Row>
+        <Grid.Col md={8}>
+          <Select
+            label={__('Expire in')}
+            className="jwt-expiration-value"
+            options={units}
+            value={expUnit}
+            onChange={e => setExpUnit(e.target.value)}
+          />
+        </Grid.Col>
+      </Grid.Row>
+      <Grid.Row>
+        <Grid.Col md={8}>
+          <FormField
+            className="jwt-expiration-value"
+            label={__('Value')}
+            disabled={expUnit === units[0].value}
+            value={expValue}
+            onChange={e => setExpValue(e.target.value)}
+            type="number"
+            inputProps={{ min: 1 }}
+          />
+        </Grid.Col>
+      </Grid.Row>
+      <Grid.Row>
+        <Grid.Col md={4}>
+          <input
```

Comment:
IMO it would be awesome to have that form inline, e,g:
`Expire in ______`           `amount ______`          
`Generate`

while the `amount` will be shown only if expiry !== 'never'

Reply:
Is there any value in this? I mean the form is self-explanatory by itself.
It could show the actual date & time of expiration, but to be honest don't know if
it is really worth of the work.

Reply:
I meant only changing it into an horizontal form, not the actual date & time of expiration,
and showing the `amount` / `value` only when needed,
it took me a few moments to understand the `value` part.. also to me `amount` sounds a better more clear

Reply:
Yeah that make sense, will do.